FordLabs/PeopleMover
# Location dropdown modified

## Issue
Resolves #450 

## What was done
- [x] Modified behavior of location drop down

## How to test
1. Verify that the new behavior math what is expected. Not everything could be implemented due to limitation of library used

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
